src/main/cs/Infinispan/HotRod/RemoteCacheManager.cs

```diff
@@ -90,15 +109,18 @@ public RemoteCacheManager(Configuration configuration, IMarshaller marshaller, b
         /// Construct an instance with default configuration and specific serializer.
         /// </summary>
         /// <param name="marshaller"></param>
+        /// <param name="argMarshaller"></param>
         /// <param name="start"></param>
-        public RemoteCacheManager(IMarshaller marshaller, bool start = true)
+        public RemoteCacheManager(IMarshaller marshaller, bool start = true, IMarshaller argMarshaller = null)
```

Comment:
Same as above. I'd say these arguments for argMarshaller are redundant. We could re-use the marshaller which was set in configuration.

Reply:
I'm not sure it could work in this way. iirc marshaller for remote exec arguments must be the jbossmarshaller.